### ISSUE TITLE
Improve Accessibility of the Login Page

### ISSUE THREAD

**Issue by JoeyHartz** [OREL-group/Project-Management#250]
**Please describe the issue.**
Currently, there are no focus indicators when users navigate through the form using a keyboard, and the contrast ratio between the text and background is insufficient. Because of this, it is challenging for users with disabilities to utilize the login page efficiently.

**How is this issue actionable?**
This issue is actionable because it provides a clear and specific goal that can be achieved through design and development changes. The issue identifies the specific accessibility issues that need to be addressed and what should be changed to do so. It is a good fit for contributors who are new to the project and want to get involved.

**Additional context**
These issues are especially critical to address for users who have visual impairments who seek to read page content, as well as users who have motor impairments and use a keyboard to navigate a page.
